src/clients/dotnet/README.md

```diff
@@ -446,3 +446,4 @@ $ dotnet clean --verbosity minimal
 $ dotnet build
 $ if ($env:TEST -eq 'true') { dotnet test } else { echo "Skipping client unit test" }
 ```
```

Comment:
might want to rebase this on main, as we normalize whitespace there now.

Reply:
Thanks!